### ISSUE TITLE
is there a pytorch option similar to one_direction_test_model for torch CycleGAN?

### ISSUE THREAD

**Issue by AllAwake** [junyanz/pytorch-CycleGAN-and-pix2pix#183]
Hello,

on GTX 1080 i'm not able to set the size of images for training higher than 384X384. I remember reading that there is a way to increase the size during the test by setting the direction - can't find this thread anymore except on the Torch CycleGAN where the option **one_direction_test_model** is suggested - is similar option available for PyTorch? Thank you!

**Comment by junyanz** [comment]
Yes, we have it in the PyTorch version. It is called [test_model](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/models/test_model.py) and you can use the flag `--model test` during test time.  See the [instructions](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix#apply-a-pre-trained-model-cyclegan) on how to apply a pretrained cyclegan model for more details.
